lib/plug/static.ex

```diff
@@ -7,12 +7,12 @@ defmodule Plug.Static do
     * `:at` - the request path to reach for static assets.
       It must be a string.
 
-    * `:from` - the filesystem path to read static assets from.
-      It must be a string, containing a file system path, or an
-      atom representing the application name where assets will
-      be served from the priv/static, or a tuple containing the
-      application name and the directory to serve assets besides
-      priv/static.
+    * `:from` - the file system path to read static assets from.
```

Comment:
> In computing, a file system (or filesystem) 

That's a bit too much. :) I don't think anyone would be confused if we pick one or the other.